ansible_collections/arista/avd/docs/pyavd.md

```diff
@@ -47,13 +44,13 @@ Feedback is very welcome. Please use [GitHub discussions](https://github.com/ari
 - Add examples
 - Add more tests (current coverage is 85%)
 - Add network state validation similar to `eos_validate_state`.
-- Add cloudvision tag integrations
-- Make pyavd the source of AVD logic and use as a dependency for the `arista.avd` Ansible collection.
+- Add CloudVision tag integrations
+- Make PyAVD the source of AVD logic and use as a dependency for the `arista.avd` Ansible collection.
 - Explore support for custom Jinja2 templates.
 
 ## Installation
 
-Install `pyavd`:
+Install the `pyavd` Python package:
```

Comment:
do we want to maintain the requirements versions in our doc going forward?

Reply:
There is no alternative right now (I tried!). The mkdocs root is the ansible collection, so we cannot read a snip from outside.

Reply:
ack - we can deal with this later once we move the doc at the top level of the repo